Sources/ArcGISToolkit/Components/Augmented Reality/Utilities/CalibrationView.swift

```diff
@@ -61,6 +61,7 @@ class WorldScaleCalibrationViewModel: ObservableObject {
     }
 }
 
+@available(visionOS, unavailable)
```

Comment:
Instead of annotating the extension, I suggest annotating the type (`CalibrationView`) itself.

Reply:
Why annotate `CalibrationView` as `unavailable` if its enclosing type is `unavailable` (`WorldScaleSceneView`)?